app/src/org/commcare/google/services/analytics/FormAnalyticsHelper.kt

```diff
@@ -0,0 +1,24 @@
+package org.commcare.google.services.analytics
+
+import org.commcare.utils.FileUtil
+import java.io.File
+import java.util.*
+
+class FormAnalyticsHelper {
+
+    var videoStartTime: Long = -1
+    var videoName: String? = null
+    var videoDuration: Long = -1
+
+    fun recordVideoPlaybackStart(videoFile: File) {
+        videoStartTime = Date().time
+        videoName = videoFile.name
```

Comment:
Good point, it will definitely be an additional slot to send the path separately. What we can do instead is just send the concatenated path with name in a single field.